R/pkg/R/DataFrame.R

```diff
@@ -1202,6 +1215,7 @@ setMethod("toRDD",
 #' Groups the SparkDataFrame using the specified columns, so we can run aggregation on them.
 #'
 #' @param x a SparkDataFrame
+#' @param ... variable(s) (character names(s) or Column(s)) to group on.
 #' @return a GroupedData
```

Comment:
I know this is a super minor and I am not even used to R. But it seems the description for `@param` starts with  a lower case and the description for `@return` starts with an upper case assuming from the other codes here and what you are trying to change.

If this is correct, we should fix this too to `a GroupedData.` to avoid.
Also, it seems your fix is trying to adds `.` at the end of the descriptions for `@return` and `@param`? In this case,  we should fix `a SparkDataFrame` to `a SparkDataFrame.` above just in order to avoid a lot of small minor PRs to fix the documentation in the future.